nncf/torch/quantization/layers.py

```diff
@@ -1064,3 +1064,24 @@ def forward(self, x):
         result = decompress(x, self._scale, self._zero_point)
         result = result.type(dtype=self.result_dtype) if self.result_dtype is not None else result
         return result
+
+
+class SymmetricWeightsDecompressor(nn.Module):
+    """
+    Applies symmetric decompression of compressed weights in the forward pass
+    """
+
+    def __init__(self, scale: torch.Tensor, result_dtype: torch.dtype = None):
+        """
+        :param scale: A scale in quantization scheme
+        :param result_dtype: (Optional) A data type that result should be cast to
+        """
+        super().__init__()
+        self.register_buffer("_scale", scale)
+        self.result_dtype = result_dtype
+
+    def forward(self, x):
+        zero_point = torch.zeros_like(self._scale)
```

Comment:
It look like a bad idea to create zero point tensor on each forward. Could you consider to introduce decompress operation for symmetric quantization?